package.json

```diff
@@ -1,26 +1,37 @@
 {
   "devDependencies": {
+    "babel-eslint": "8.0.2",
+    "eslint-config-prettier": "2.7.0",
+    "eslint-plugin-prettier": "2.3.1",
+    "eslint-plugin-react": "7.4.0",
+    "eslint": "4.11.0",
     "husky": "0.14.3",
     "lerna": "2.5.1",
-    "lint-staged": "4.3.0",
-    "prettier": "1.8.1"
+    "lint-staged": "5.0.0",
+    "prettier": "1.8.2"
   },
   "lint-staged": {
-    "*.{js,ts}": [
-      "yarn lint:script",
-      "git add"
-    ]
+    "*.{js,jsx}": ["eslint --ignore-path .prettierignore --fix", "git add"],
+    "*.json": ["prettier --write", "git add"],
```

Comment:
Why not using ` "*.json": ["yarn fix:config", "git add"]` here?

Reply:
because this one will run/fix all the `*.json` files that are included in the project instead of just the staging ones..

Reply:
Ah, ok! Didn't knew that.